portfolio/src/main/webapp/index.html

```diff
@@ -73,9 +73,23 @@ <h1>Blog</h1>
         </div>
         <div id="confirm"></div>
       </div>
+
+      <div id="blogreplycontent">
+        <form id="my-form" enctype="multipart/form-data">
+          <textarea id="sender" rows="1" cols="20" placeholder="Nickname"></textarea> 
+          <br>
+          <textarea id="text-input" rows="6" cols="60" placeholder="What's on your mind..."></textarea>
+          <br>
+          <br>
+          Upload an image:
+          <input type="file" id="image" name="image">
+        </form>
+        <button onclick="sendAndGet()">Post</button> 
+      </div>
+      
     </div>
  
-    <div id="Information" class="tabcontent">
+    <div id=" formation" class="tabcontent">
```

Comment:
This must have happened on accident, will fix.

Reply:
Ah I like the fact that you self checked the code:)